recipes/azure-mgmt-synapse/recipe.yaml

```diff
@@ -0,0 +1,47 @@
+schema_version: 1
+
```

Comment:
I have not seen this in other PRs
```suggestion
```

Reply:
This was auto-generated by `greyskull pypi --use-v1-format`.